### DISCUSSION TITLE
personal-development-aban-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#61] [Announcements]
# personal-development-aban-1403/

ماه آبان هم به پایان رسید و وقتشه که گزارش آبان رو منتشر کنم. این ۲۲مین گزارشی هست که منتشر می‌کنم. واقعیتش اولی که نوشتن این گزارش‌ها رو شروع کردم، فکر نمی‌کردم که بخوام و بتونم تا تقریباً دو سال این کار رو ادامه بدم؛ ولی شد و راضیم.
ماه آبان برام ماه پر از شلوغی بود. به نظرم تو ماه‌های بعد سراغ فعالیت جدید نباید برم و قطعاً یکی دو تا از فعالیت‌های خارج برنامه‌ام رو هم باید کمتر کنم.

https://aprd.ir/personal-development-aban-1403/

**Comment by iamamohsen** [comment]
سلام امیر جان، امیدوارم حالت خوب باشه
خیلی خوشحالم که نوشته‌هات رو می‌خونم؛ این گزارش ها من رو همیشه به فکر فرو می‌برن. همینطور دقت و ظرافت و طنز خاصی که تو نویسندگی‌ت به کار می‌بری واقعا تحسین‌برانگیزه. مثل همیشه، گزارش ماهیانه‌ات الهام‌بخش بود و نشون می‌ده چقدر با برنامه و هدفمند پیش می‌ری. ممنون که با ما این مسیر رو به اشتراک می‌ذاری. امیدوارم در آینده هم این روال رو ادامه بدی.

**Reply by pourmand1376** [reply]
سلام محسن جان. ممنون از لطف و توجه‌ات. 

خوشحالم که گزارش‌هام باعث میشه به فکر فرو بری؛ البته فکر نکنم انقدرها هم هدف‌مند و با برنامه باشم. از نظر خودم که این‌طور نیستم. ولی از بیرون شاید این‌طور به نظر بیاد.
